docs/lib/datastore/fragments/android/getting-started/60_saveSnippet.md

```diff
@@ -0,0 +1,30 @@
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Post post = Post.builder()
+    .title("Create an Amplify DataStore app")
+    .build();
+
+Amplify.DataStore.save(post,
```

Comment:
This save is only local right, what will happen if there is an error when sending the mutation, for example AuthZ issues?